plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/Deployment.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.explorer;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.openshift.restclient.model.IPod;
+import com.openshift.restclient.model.IService;
+import com.openshift.restclient.model.route.IRoute;
+
+/**
+ * A deployment is the collection of resources
+ * that makes up an 'application'
+ * 
+ * @author jeff.cantrill
+ *
+ */
+public class Deployment {
+
+	private IService service;
+	private List<IRoute> routes = new ArrayList<>();
+	private List<IPod> pods;
```

Comment:
maybe use Sets instead of lists? so that you avoid potential duplicates when doing add(pod)